amethyst_state/src/game_data.rs

```diff
@@ -213,7 +226,7 @@ impl<'a, 'b> GameDataBuilder<'a, 'b> {
     {
         bundle
             .build(&mut self.disp_builder)
-            .map_err(|err| Error::Core(err))?;
+            .map_err(|err| Error(err.0,err.1))?;
```

Comment:
does this compile? `error-chain` [docs](https://docs.rs/error-chain/0.12.0/error_chain/example_generated/struct.Error.html) say the second field is private.

The returned error type changes from `amethyst::error::Error(core::Error(..))` to `core::Error(..)`.  `amethyst::error::Error` does implement `From<core::Error>`, so most apps that simply use `.with_bundle(..)?` can live with a straight recompile, but if any of them actually match over the error type will break. So at minimum we need a changelog entry for this.

also, missing a space after the comma.